modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/service/impl/ClientExtensionEntryLocalServiceImpl.java

```diff
@@ -518,6 +547,57 @@ private ClientExtensionEntry _startWorkflowInstance(
 			clientExtensionEntry, serviceContext, new HashMap<>());
 	}
 
+	private void _undeployClientExtensionEntryClusterWide(
```

Comment:
shouldn't we remove the [@Clusterable annotation here](https://github.com/liferay-frontend/liferay-portal/pull/4548/commits/84a19ea70b0b32381eecc802c13b820a27e852ac#diff-a8af7631485cb3179a7ea0e6eba70e2683bcb9f9709d93aa0a3bb390abfc2dacR324)?